tests/lambda_endpoints/test_indexer_lambda.py

```diff
@@ -185,13 +185,17 @@ def test_batch_job_event(test_engine, write_to_s3, events_client, set_env):
                     "123456789012.dkr.ecr.us-west-2.amazonaws.com/" "codice-repo:latest"
                 ),
                 "command": [
```

Comment:
I'm not sure I follow this command and what it is doing.
I read it as: process the SWE instrument at l1b, but the filepath is for SWAPI l1, and the dependency is SWE l0.

Reply:
good catch. I was copy pasting lot yesterday with fix. It was remain from that. will fix it.